<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlutoFullAssessor</class>
 <widget class="QMainWindow" name="PlutoFullAssessor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>344</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>344</width>
    <height>380</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Bahnschrift Light</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>PLUTO Full Assessment</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QPushButton" name="pbTestDevice">
        <property name="font">
         <font>
          <family>Bahnschrift Light</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Test Device</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbSubject">
        <property name="font">
         <font>
          <family>Bahnschrift Light</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Select the subject to assess</string>
        </property>
        <property name="text">
         <string>Select Subject</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Subject Type:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbSubjectType">
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Healthy</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Stroke</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Limb:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbLimb">
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Left</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Right</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pbSetLimb">
        <property name="text">
         <string>Set Limb</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gbMechanisms">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="title">
         <string>Mechanisms</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QRadioButton" name="rbWFE">
             <property name="text">
              <string>Wrist Flexion-Extension</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbFPS">
             <property name="text">
              <string>Forearm Pronation-Supination</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbHOC">
             <property name="text">
              <string>Hand Opening-Closing</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbCalibrate">
        <property name="font">
         <font>
          <family>Bahnschrift Light</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Calibrate Mechanism</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+S, Ctrl+S</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbRomAssess">
        <property name="font">
         <font>
          <family>Bahnschrift Light</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Assess ROM</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbDiscReachAssess">
        <property name="text">
         <string>Discrete Reaching</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbPropAssess">
        <property name="text">
         <string>Proprioceptive Assessment</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbForceCtrlAssess">
        <property name="text">
         <string>Force Control Assessment</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <family>Cascadia Mono</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
